app/controllers/comments_controller.rb

```diff
@@ -16,4 +16,14 @@ def destroy
     @comment.destroy
     redirect_to(post_path)
   end
+
+  def toggle_like_comment
+    @comment = Comment.find(params[:comment_id])
+    if current_user.liked? @comment
+      @comment.unliked_by current_user
+    else
+      @comment.liked_by current_user
+    end
+    redirect_to("/posts/#{@comment.post_id}")
```

Comment:
probably something to note for later, but we probably don't want to refresh the page when liking a comment. leave it as a scalability improvement ;)